README.md

```diff
@@ -224,6 +224,7 @@ docker stack deploy -c contrib/swarm/docker-stack.yml mattermost
 * Do not modify the Listen Address in Service Settings.
 * Rarely `app` container fails to start because of "connection refused" to
   database. Workaround: Restart the container.
+* Don't update to release-5.32 though, it's broken: #489
```

Comment:
Would something like this work?

```suggestion
* Upgrading to ``release-5.32`` fails because PostgreSQL 10+ is required. We don't currently have a timeline for the fix, and as an intermediate option we recommend trying out [Mattermost Omnibus](https://docs.mattermost.com/install/mattermost-omnibus.html) which supports PostgreSQL 10+.
```

Reply:
I think it is much better and very clear.

Reply:
Thanks, I don't have access to push the edits here.

Reply:
@nvjacobo For those who don't want to use Omnibus (and we don't have a guide to migrate from docker to Omnibus), maybe we could also add a link to this script as a second option (which was created by our support team): https://github.com/mattermost/mattermost-docker/issues/489#issuecomment-790277661.

Reply:
I updated the script a bit to make it easier to use. It's a temporary workaround here.

@tuxity & @GuidoDr outlined the process of how best to do this, I merely scripted it!